west.yml

```
@@ -72,7 +72,7 @@ manifest:
       revision: b52fdbf4b62439be9fab9bb4bae9690a42d2fb14
       path: modules/hal/st
     - name: hal_stm32
-      revision: 68bfdabe97aa60934e1519e98593abaa9745501e
+      revision: pull/95/head
```

Could go with following, now:
```suggestion
      revision: f8ff8d25aa0a9e65948040c7b47ec67f3fa300df
```

---

Thanks, I have just done that.